### ISSUE TITLE
Is it possible to implement/use a custom Viewport other than Viewport provided by flutter

### ISSUE THREAD

**Issue by natintosh** [GeekyAnts/infinite-carousel-flutter#28]
Would be nice if I don't have to constrain the height/width if I'm using either Axis.horizontal or Axis.vertical. Is it possible to have a behaviour like SingleChildScrollView combined with Row?

**Comment by manuindersekhon** [comment]
Hi @natintosh, this may be achieved with `SliverList` instead of `SliverFixedExtentList` that is currently used in the package code. I've explained about this in more details in this issue (https://github.com/GeekyAnts/infinite-carousel-flutter/issues/17#issuecomment-1756484873). Please check it out once if this helps. Thanks.

**Comment by natintosh** [comment]
Yes @manuindersekhon I agree using SliverList fixed the issue with Fixed Extent on the main scroll axis and allow dynamic size on the main scroll axis but soon the cross axis we still need to specify a fixed size (height/widget).
